Sources/Sentry/SentryANRTrackingIntegration.m

```diff
@@ -70,19 +63,32 @@ - (BOOL)shouldBeDisabled:(SentryOptions *)options
 
 - (void)uninstall
 {
-    [self.tracker stop];
+    [self.tracker removeListener:self];
 }
 
 - (void)anrDetected
 {
-    [self.appStateManager
-        updateAppState:^(SentryAppState *appState) { appState.isANROngoing = YES; }];
+    NSString *message =
+        [NSString stringWithFormat:@"Application Not Responding for at least %li ms.",
+                  (long)(self.options.anrTimeoutInterval * 1000)];
+
+    NSArray<SentryThread *> *threads =
+        [SentrySDK.currentHub.getClient.threadInspector getCurrentThreadsWithStackTrace:YES];
+
+    SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
+    SentryException *sentryException = [[SentryException alloc] initWithValue:message
+                                                                         type:@"App Hanging"];
+    sentryException.mechanism = [[SentryMechanism alloc] initWithType:@"anr"];
```

Comment:
`h`: How do ANR errors group in Sentry?

Reply:
Im not familiar with the grouping process in Sentry, but I saw it doing it. Even with different timespans in the description.